pkg/amt/setupandconfiguration/service.go

```diff
@@ -15,6 +16,17 @@ import (
 
 const AMT_SetupAndConfigurationService = "AMT_SetupAndConfigurationService"
 
+type UnprovisionResponse struct {
```

Comment:
We haven't been doing nested structs for these response, you mind breaking it out to stay consistent?